### PR TITLE
[AIRFLOW-4168] Create Google Cloud Video Intelligence Operators

### PR DESCRIPTION
**Pull request by antonimaciej**
Make sure you have checked _all_ steps below.

### Jira

https://issues.apache.org/jira/browse/AIRFLOW-4168

### Description

Create Video Intelligence operators:
CloudVideoIntelligenceDetectVideoLabelsOperator,
CloudVideoIntelligenceDetectVideoExplicitContentOperator,
CloudVideoIntelligenceDetectVideoShotsOperator

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
CloudVideoIntelligenceHookTestCase, CloudVideoIntelligenceOperatorsTestCase

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  
### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

File: docs/howto/operator/gcp/video.rst [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+"""""""""
+
+Some arguments in the example DAG are taken from the OS environment variables:
+
+.. literalinclude:: ../../../../airflow/contrib/example_dags/example_gcp_video_intelligence.py
+      :language: python
+      :start-after: [START howto_operator_video_intelligence_os_args]
+      :end-before: [END howto_operator_video_intelligence_os_args]
+
+Input uri is an uri to a file in google cloud storage
```
Comment (mik-laj) [inline docs/howto/operator/gcp/video.rst]:
```suggestion
Input uri is an uri to a file in Google Cloud Storage
```

File: airflow/contrib/example_dags/example_gcp_video_intelligence.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that demostrated operators for the Google Cloud Video Intelligence service in the Google
```
Comment (kaxil) [inline airflow/contrib/example_dags/example_gcp_video_intelligence.py]:
```suggestion
Example Airflow DAG that demonstrates operators for the Google Cloud Video Intelligence service in the Google
```

File: airflow/contrib/example_dags/example_gcp_video_intelligence.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+Cloud Platform.
+
+This DAG relies on the following OS environment variables
```
Comment (kaxil) [inline airflow/contrib/example_dags/example_gcp_video_intelligence.py]:
```suggestion
This DAG relies on the following OS environment variables:
```

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
@antonimaciej - just the two typos from @kaxil and I think we are good to go.

**Pull request merged**